src/main/java/by/custom/utilcalculator/service/MessagesCreator.java

```diff
@@ -192,19 +191,23 @@ private String checkForGasolineAuoAge(UserProgress userProgress) {
         return resultString;
     }
 
-    private String checkForElectricAutoAge(UserProgress userProgress) {
+    private String checkForElectricAutoAge(final UserProgress userProgress) {
         return userProgress.getCarAge() == null ? getAgeAuto() : getResultAndFarewell(userProgress);
     }
 
-    private String stringBuilderAppender(String... strings) {
+    private String stringBuilderAppender(final String... strings) {
         StringBuilder stringBuilder = new StringBuilder();
         for (String string : strings) {
             stringBuilder.append(string);
         }
         return stringBuilder.toString();
     }
 
-    private String trimFirstAndLastLetters(String toTrim) {
+    public String getSummaryAnswer(final UserProgress userProgress) {
+        return getCountryOrigin(userProgress);
```

Comment:
We have to discuss it. For me looks strange that `getSummaryAnswer` returns only `getCountryOrigin`.

Reply:
Yes, it really looks wrong. But I don't know ho to do it better. 
The logic of building message for user works from getCountryOrigin method. It starts to work here: https://c2n.me/4jXc5Uk - and runs until the message completed. 
Probably, we should rename getCountryOrigin into "getMessageForUser" or something like that? 
I guess, it is more naming problem.